koku/api/dataexport/models.py

```diff
@@ -21,6 +21,7 @@ class DataExportRequest(models.Model):
     status = models.CharField(max_length=32, choices=STATUS_CHOICES, default=PENDING)
     start_date = models.DateField(null=False)
     end_date = models.DateField(null=False)
+    bucket_name = models.CharField(max_length=63)
```

Comment:
I think this needs a False null/blank setting because it should be a required value.

Reply:
Null and blank default to False.
https://docs.djangoproject.com/en/2.2/ref/models/fields/#null

```
Field.null¶
If True, Django will store empty values as NULL in the database. Default is False.
```
```
Field.blank¶
If True, the field is allowed to be blank. Default is False.
```